css/style.css

```diff
@@ -88,10 +88,17 @@ body {
 .content-divider {
   height: 3px;
   width: 30%;
-  background-color: var(--secondary-color);
+  background-color: var(--tertiary-color);
   margin: 10px auto;
 }
 
+.header-text-component {
+  text-align: center;
+  width: 60%;
+  margin-left: auto;
```

Comment:
It would be better to use shorthand here such as `margin: 0 auto;`

Reply:
done